src/Paper.types.js

```diff
@@ -45,6 +47,7 @@ const TYPES: Types = {
   [CONSTANTS.Layer]: (props, paper) => new paper.Layer(props),
   [CONSTANTS.Group]: (props, paper) => new paper.Group(props),
   [CONSTANTS.Path]: (props, paper) => new paper.Path(props),
+  [CONSTANTS.Raster]: (props, paper) => new paper.Raster(props),
```

Comment:
Constructor takes a certain parameters: http://paperjs.org/reference/raster/#. You need to map the prop object to the correct constructor usage.

Reply:
My mistake, I see that Paper supports size as a option.